doc/source/releases/15.05.rst

```diff
@@ -0,0 +1,348 @@
+* Add option to track all automated changes to the integrated tool panel.
+  10bb492_
+* Make tool version explicit in all distribution tool - thanks to Peter Cock. 
+  `Pull Request 14`_. 
+* Relocate the external metadata setting script. `Pull Request 7`_
+* Parameterize script used to pull new builds from the UCSC Browser.
+  e4e5df0_
+* Enhance jobs and workflow logging to report timings. 06346a4_
+* Add debug message for dynamic options exceptions. `Pull Request 91`_
+* Remove demo sequencer app. 3af3bf5_
+* Tweaks to the Pulsar's handling of async messages. `Pull Request 109`_
+* Return more specific API authentication errors. 71a64ca_
+* Upgrade Python dependency sqlalchemy to 1.0.0. d725aab_
+* Upgrade Python dependency amqp to 1.4.6. e09761e_
+* Upgrade Python dependency kombu to 3.0.24. 8d3c531_
+* Upgrade JavaScript dependency raven.js to 1.1.17. bcd1701_
+
+Fixes
+-------------------------------
+
+* During the 15.05 development cycle dozens of fixes were pushed to the
+  ``release_15.03`` branch of Galaxy. These are all included in 15.05 and
+  summarized `here
+  <https://github.com/galaxyproject/galaxy/compare/v15.03...release_15.03>`__
+  (with special thanks to Björn Grüning and Marius van den Beek).
+* Fix race condition that would occasionally prevent Galaxy from starting
+  properly. `Pull Request 198`_, `Trello <https://trello.com/c/WVlaLsOh>`__
+* Fix scatter plot API communications for certain proxied Galaxy instances -
+  thanks to @yhoogstrate. `Pull Request 89`_
+* Fix bug in collectl_ job metrics plugin - thanks to Carrie Ganote. 
+  `Pull Request 231`_
+* Fix late validation of tool parameters. `Pull Request 115`_
+* Fix ``fasta_to_tabular_converter.py`` (for implicit conversion) - thanks to
+  Peter Cock. `Pull Request 11`_
+* Fix to eliminate race condition by collecting extra files before declaring
+  dataset's OK. `Pull Request 48`_
+* Fix setting current history for certain proxied Galaxy instances - thanks
+  to @wezen. 6946e46_.
+* Fix typo in tool failure testing example - thanks to Peter Cock.
+  `Pull Request 18`_.
+* Fix Galaxy to default to using SSL for communicating with Tool Sheds.
+  0b037a2_
+* Fix data source tools that do not have SSL to open in ``_blank`` window.
+  `Pull Request 17`_
+* Fix to fallback to name for tool parameters without labels.
+  `Pull Request 189`_, `Trello <https://trello.com/c/Y2xbXqzZ>`__
+* Fix to remove redundant version ids in tool version selector.
+  `Pull Request 244`_
+* Fix for downloading metadata files. `Pull Request 234`_
+* Fix for history failing to render if it contains more exotic dataset 
+  collection types. `Pull Request 196`_
+* Fixes for BaseURLToolParameter. `Pull Request 247`_
+* Fix to suppress pysam binary incompatibility warning when using datatypes
+  in ``binary.py``. `Pull Request 252`_
+* Allow a tool data table to declare that duplicate entries are not
+  allowed. `Pull Request 245`_
```

Comment:
Yes, yes it does. Though somehow it was merged after the deadline for new features :).

I'll move this.